### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Stanenko**
- [DEMO LINK](https://Stanenko.github.io/layout_stars/)
- [TEST REPORT LINK](https://Stanenko.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,42 @@
...
+  background-image: url('../src/images/star-active.svg');
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url('../src/images/star-active.svg');
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url('../src/images/star-active.svg');
+}
```
Comment (volodymyr-soltys97) [inline src/style.css:42]: These selectors must be separated by commas

File: src/style.css [hunk 2]
```diff
@@ -1 +1,42 @@
...
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../src/images/star.svg);
```
Comment (volodymyr-soltys97) [inline src/style.css:14]:
```suggestion
  background-image: url(./images/star.svg);
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,42 @@
...
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url('../src/images/star-active.svg');
```
Comment (volodymyr-soltys97) [inline src/style.css:25]:
```suggestion
  background-image: url('./images/star-active.svg');
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's make your code better

File: src/style.css [hunk 4]
```diff
@@ -1 +1,47 @@
...
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1,
+.stars__star:nth-child(-n + 1) {
```
Comment (anastasiiavorobiova) [inline src/style.css:25]:
```suggestion
.stars__star:nth-child(-n + 1),
stars--2,
.stars__star:nth-child(-n + 2) ,
.... {
```
Consider grouping your selectors with commas

**Pull request closed without merging**